### ISSUE TITLE
Windows development using CMake

### ISSUE THREAD

**Issue by kosumosu** [Swartz27/xray-16#1575]
**Is your feature request related to a problem? Please describe.**
Windows development is now only possible using Visual Studio. There's much better options like CLion for example. But currently CMake is only properly working on linux.

**Describe the solution you'd like**
I would suggest to move fully to CMake, even on Windows, because it will allow developers to choose IDE's they like. Who want to stay with VS - can still generate VS project for themselves.

Now I have to manually tune `CMakeLists.txt` to open in CLion without errors.

Just a side note:
I also thing SDL include path is not set anywhere in `CMakeLists.txt`, it's only luck that it works on linux.

**Comment by Xottab-DUTY** [comment]
I fully agree on this, we plan to move to CMake on Windows, and @Masterkatze is currently working on it.

**Comment by Kirilliriks** [comment]
I agree with this suggestion, Clion is an excellent tool
